### PR TITLE
Bug Fix - 367

### PR DESCRIPTION
**Pull request by mbowcut2**
@kclem This is a fix for bug: https://github.com/pinellolab/CRISPResso2/issues/367

It looks like the problem is when all reads are aligned to 'Prime-edited' and none to 'Reference', then
`refs_names_for_pe[0]` is `Prime-edited` when it is expected to be `Reference`. This causes errors when the wrong sequence is used for `colnames` and the `totals` row.

I changed line 4566 and 4567, which fixed the bug.

I also saw other references to `refs_names_for_pe[0]` in lines 4569-4572 that I assumed were erronious, and changed those as well.

### CODE REVIEW

File: CRISPResso2/CRISPRessoCORE.py [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ def rreplace(s, old, new):
                     auto_fastq_r2 = ""
                     files_to_remove.append(auto_fastq_r1)
                     sb.call('samtools view -h ' + args.bam_input + ' ' + args.bam_chr_loc + ' | head -n ' + str(number_of_reads_to_consider+100) + ' | samtools bam2fq -  2> /dev/null > ' + auto_fastq_r1, shell=True)
-
+
```
Comment (Colelyman) [inline CRISPResso2/CRISPRessoCORE.py:1241]: I would recommend deleting these spaces/tab, otherwise this looks good!

**Comment by kclem** [comment]
Thanks, @mbowcut2 I had done it this way because sometimes the 0'th amplicon isn't named 'Reference'. Try running with the -an parameter with prime editing. There may be a way to load the 0'th amplicon info into ref_names_for_pe on line https://github.com/edilytics/CRISPResso2/pull/35/files#diff-6043decbacd7b8775dc4013839dc08bcb168ec37f1031a60e8ee89291d23c06eL4550

**Comment by Colelyman** [comment]
@kclem that is a great point that I hadn't thought of either... do you guys think that `ref_names[0]` would do the trick? Because in PE mode the `Prime-edited` amplicon is always going to be based on the 0th amplicon, right? Would you ever supply multiple amplicons in PE mode?

**Comment by mbowcut2** [comment]
@kclem Okay, what about this? We just use `ref_names[0]`. That fixes the bug, and it works with `-an`. @Colelyman could you check the reports?

**Comment by Colelyman** [comment]
@mbowcut2 I just talked with @kclem and he is going to verify that this works and that there isn't anything else that we are missing anything.

It does look like the `'Prime-edited` amplicon is going to refer to the first amplicon (see: https://github.com/edilytics/CRISPResso2/blob/mckay/issue-367/CRISPResso2/CRISPRessoCORE.py#L1373C76-L1373C95 ), which could be something that we address in another PR. I'm going to do some more thorough testing to verify this...

**Comment by Colelyman** [comment]
@kclem I have been testing this and from what I can tell the pegRNA sequence is always going to align to the first amplicon (even if it aligns better to the second amplicon). I think that this is a separate issue that should be addressed in a different PR, and that we can at least get these changes in. What do you think?

**Comment by Colelyman** [comment]
@kclem this has been tested with reads that align to both Reference and Prime-edited and also only reads that align to Reference. Great work @mbowcut2!

**Pull request merged**